README.adoc

```diff
@@ -212,6 +212,33 @@ refute(actual.empty?)
 refute_empty(actual)
 ----
 
+=== Assert Match [[assert-match]]
+
+Use `assert_match` if expecting matcher regex to match actual object.
+
+[source,ruby]
+----
+# bad
+assert(pattern.match?(actual))
```

Comment:
As there's no "expected" here I'm not sure whether "actual" is the best name in this context. Maybe just "thing"/"object"?

Reply:
Object sounds good to me. I'll change it to object.

Reply:
@bbatsov : Changed `expected` -> `object`. Could you please review it?